internal/backend/state.go

```diff
@@ -466,17 +466,21 @@ func (state *State) updateMessageID(oldID, newID string) error {
 	return nil
 }
 
+func (state *State) cleanupConnMetadata() error {
```

Comment:
```suggestion
func (state *State) deleteConnMetadata() error {
```

i'd prefer this as it's more clear what "clean up" actually means.

Alternatively, just to make me happy (sorry!), consider this instead:
```suggestion
func (state *State) cleanUpConnMetadata() error {
```

Reply:
Changed to `deleteConnMetadata`